build.gradle

```diff
@@ -363,3 +364,16 @@ task getDockerImageTag {
         println("${project.properties['dockerImageTag']}")
     }
 }
+
+test {
```

Comment:
Forgot to mention, can we do "Limiting retry to CI builds"?

https://github.com/gradle/test-retry-gradle-plugin#limiting-retry-to-ci-builds

For local runs, often I'm working on a legitimately failing test